hack/util.sh

```diff
@@ -458,10 +458,14 @@ function install_registry() {
 }
 readonly -f install_registry
 
-function wait_for_registry() {
+function is_registry_ready() {
```

Comment:
NACK to prefixing it with anything. The correct format in correct files (I'm working on it ...) is `os::<whatever>::internal::<funcname>`